TicketBookingAPI/Controllers/FlightController.cs

```diff
@@ -99,6 +99,18 @@ public async Task<IActionResult> GetFlightByRequest([FromQuery] FlightRequest fl
             
             return Ok(await flightservice.GetFlightAsync(flightrequest));
         }
+        
+        [HttpGet("GetflightByRequestWithPaging")]
+        public async Task<IActionResult> GetFlightByRequestPaging([FromQuery] FlightRequest flightrequest, [FromQuery] RequestParam request)
```

Comment:
should combine FlightRequest and RequestParam into one model. Use inherit

Reply:
fixed